css/styles.css

```diff
@@ -0,0 +1,543 @@
+
+* {
+  margin: 0;
+  padding: 0%;
+  box-sizing: border-box;
+}
+
+body {
+  position: relative;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.wrapper {
+  height: 5410px;
+  max-width: 810px;
+  background: #fff;
+  display: grid;
+  grid-template-rows: 0.04fr 0.32fr 0.32fr 0.32fr;
+}
+
+.container-homepage {
+  /* position: absolute; */
+
+  background-image: url("/images/header-shapes-mobile.png");
+  height: 604.0000610351562px;
+  max-width: 100%;
+
+  /*
+  left: calc(50% - 375px / 2);
+  top: 92px;
+  */
+
+  padding: 114px, 24px, 114px, 24px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+}
+
+.header-2 {
+  height: 48px;
+  max-width: 100%;
+  margin-left: 24px;
+  margin-right: 24px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.my-logo {
+  width: 100px;
+  height: 40px;
+  color: #6070ff;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  text-decoration: none;
+}
+
+.bar {
+  background-color: #fff;
+  border: 0;
+  cursor: pointer;
+}
+
+.navbar {
+  height: 40px;
+  width: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 8px;
+}
+
+li a i {
+  position: absolute;
+  left: 16.67%;
+  right: 16.67%;
+  top: 29.17%;
+  bottom: 70.83%;
+  color: #6070ff;
+  width: 16px;
+}
+
+.menu-bar button i {
+  height: 12px;
+  width: 18px;
+
+  /*
+  left: 3px;
+  top: 6px; */
+
+  color: #6070ff;
+}
+
+li {
+  list-style: none;
+}
+
+/* Styling anchor tags */
+a {
+  text-decoration: none;
+}
+
+li a {
+  text-decoration-style: none;
+  position: relative;
+}
+
+.section-home {
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.name-welcome {
+  color: #17284d;
+  font-style: normal;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  height: 150px;
+  width: 327px;
+  display: flex;
+  align-items: center;
+  margin-top: 128px;
+  margin-bottom: 12px;
+}
+
+.description-of-me {
+  color: #344563;
+  margin-bottom: 12px;
+}
+
+.lets-connect-statement {
+  width: 125px;
+  height: 24px;
+  color: #7f8cff;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.contact-with-social {
+  color: #7f8cff;
+}
+
+.social-networks li {
+  display: inline-block;
+}
+
+.social-network {
+  text-decoration: none;
+  padding-right: 30px;
+}
+
+li .social-network i {
+  color: grey;
+  padding-right: 10px;
+  font-size: 20px;
+  font-weight: 700;
+}
+
+.social-networks .social-network i:hover {
+  color: #6070ff;
+  box-shadow: 0.001px 8px 16px rgb(64, 83, 252, 0.24);
+  border-radius: 8px;
+}
+
+/* Works and About-me section */
+.section-works {
+  margin: 16px 16px;
+}
+
+.card-works {
+  height: 220px;
+  max-width: 100%;
+  border-radius: 8px;
+  margin-bottom: 350px;
+}
```

Comment:
- [ ] Please as demanded by the project requirements. Kindly use CSS flexbox to lay out the contents of your cards.